.gitignore

```diff
@@ -86,15 +87,15 @@ __pycache__/
 target/
 /test/
 /tmp/
-TAGS
-TAGS.emacs
-TAGS.vi
+tags*
+TAGS*
 \#*
 \#*\#
 config.mk
 config.stamp
 keywords.md
 lexer.ml
+Session.vim
```

Comment:
It might make sense to add this to your personal `~/.gitignore` (that is where I have e.g. `*~`, which rules out emacs backup files). But I don't really mind putting it here also.

Reply:
I'll do that, I forgot that's a thing you could do. I'll leave it here though because it doesn't hurt.